source/codegen/metadata/nidaqmx/enums.py

```diff
@@ -685,21 +686,24 @@
         'values': [
             {
                 'documentation': {
-                    'description': ' Sensor is a full bridge. If you set DAQmx_AI_Excit_UseForScaling to TRUE,  NI-DAQmx divides the measurement by the excitation value. Many sensors scale  data to native units using scaling of volts per excitation.'
+                    'description': ' Sensor is a full bridge. If you set DAQmx_AI_Excit_UseForScaling to True,  NI-DAQmx divides the measurement by the excitation value. Many sensors scale  data to native units using scaling of volts per excitation.',
+                    'python_description': 'Sensor is a full bridge. If you set **ai_excit_use_for_scaling** to True, NI-DAQmx divides the measurement by the excitation value. Many sensors scale data to native units using scaling of volts per excitation.'
                 },
                 'name': 'FULL_BRIDGE',
                 'value': 10182
             },
             {
                 'documentation': {
-                    'description': ' Sensor is a half bridge. If you set DAQmx_AI_Excit_UseForScaling to TRUE,  NI-DAQmx divides the measurement by the excitation value. Many sensors scale  data to native units using scaling of volts per excitation.'
+                    'description': ' Sensor is a half bridge. If you set DAQmx_AI_Excit_UseForScaling to True,  NI-DAQmx divides the measurement by the excitation value. Many sensors scale  data to native units using scaling of volts per excitation.',
```

Comment:
This includes unwanted changes to the C/CVI descriptions (e.g. `TRUE` -> `True`). When that is resolved in the scrapigen PR, please update this PR.

Reply:
Updated the expected enums.py from Scrapigen https://github.com/ni/nidaqmx-python/pull/215